src/bower.json

```diff
@@ -7,5 +7,10 @@
     "leaflet": "^0.7.3",
     "multiple-select": "~1.1.0",
     "ramda": "0.8.0"
+  },
+  "overrides": {
+    "leaflet": {
+      "ignore": true
```

Comment:
True, and it was being included. The bigger issue is that img paths get mangled by the static files copy so things end up in the wrong place and fixing that would require an effort similar to the multipleselect library, but more complicated since there are more images.